docs/design/classes.md

```diff
@@ -2100,10 +2101,75 @@ that "`T` is comparable to `U` if they have the same field names in the same
 order, and for every field `x`, the type of `T.x` implements `ComparableTo` for
 the type of `U.x`."
 
+## Alternatives considered
+
+-   [#257: Initialization of memory and variables](https://github.com/carbon-language/carbon-lang/pull/257)
+
+    -   [Require compile-time-proven initialization](/proposals/p0257.md#require-compile-time-proven-initialization)
+    -   [C/C++ uninitialized](/proposals/p0257.md#cc-uninitialized)
```

Comment:
```suggestion
    -   [C and C++ uninitialized](/proposals/p0257.md#cc-uninitialized)
```
Very minor, but "C/C++" irritates some people in the C++ community so we may as well avoid it here.

Reply:
I'll send you a PR separately for this.